lib/src/emitter.dart

```diff
@@ -71,17 +71,19 @@ class DartEmitter extends Object
   ///
   /// May specify an [Allocator] to use for symbols, otherwise uses a no-op.
   DartEmitter(
-      [this.allocator = Allocator.none,
+      {this.allocator = Allocator.none,
       bool? orderDirectives = false,
-      bool? useNullSafetySyntax = false])
+      bool? useNullSafetySyntax = false})
```

Comment:
I can't remember if we had already discussed this - why do these `bool` need to be nullable if we have a default?

Reply:
Agreed! run w/ the breaking change!

Reply:
For unsound apps, but I am OK with making this change.

Reply:
FWIW I'd rather do `useNullSafetySyntax = `(useNullSafetySyntax as dynamic) ?? false;` than to put the nullable type in the signature.

Reply:
@natebosch then I have to cast it back to bool also... what about just not supporting unsound apps that pass an explicit null? Imo for this package that would be fine.

Reply:
yeah, usage is low and we can reasonably fix opted out usage before publishing with an expanded constraint. I'm in favor of the behavior change.